<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysing conduit network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInputFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Amira file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>1251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAnalyse">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>1311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>831</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>1251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sfactor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_sfactor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Shrinkage factor</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_traverse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>241</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_sphere">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Starting sphere</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_centre">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Centre (x,y,z)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_x0">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_y0">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_radius">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_radius">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_um2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(um)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_speed">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>13</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_um3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(um/min)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Average speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pow">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cosine power</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pow">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ntrials">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>300</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ntrials">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of trials</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_savepaths">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Save paths</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_npaths">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of paths</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_npaths">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>350</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_z0">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Coeff. of var.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CV">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_jump">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Jumping</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_nojump">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No jumping</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_jump</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_ejump">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Edge jumps</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_jump</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_vjump">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertex jumps</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_jump</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_jumpprob">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Jump prob factor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_jumpprob">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_CM">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CM estimation</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_statistics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_len_limit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_len_diam_limit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>length/diameter</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_limits</string>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="lineEdit_len_diam_limit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_len_limit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>length</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_limits</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_limits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Vessel limits</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_no_limit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>no limit</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_limits</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_dlen">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vessel length box width (um)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dlen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ddiam">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vessel diameter box width (um)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_histogram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Histogram parameters</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ddiam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_um1_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(um)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_modes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_clean">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove unconnected fibres</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_mode</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect dead ends</string>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_mode</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_cm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Estimate CM</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string>buttonGroup_mode</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_um1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(um)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_maxlen">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxlen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum length (unscaled)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_clean">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(creating clean.am)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mode of operation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statistics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Network statistics</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_deadend_radius">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dead end radiius</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_deadend_radius">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_um1_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(um)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAnalyse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>conduit_analyser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_savepaths</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>savepaths_checkbox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>outputFileSelecter()</slot>
  <slot>inputFileSelecter()</slot>
  <slot>conduit_analyser()</slot>
  <slot>join_checkbox()</slot>
  <slot>savepaths_checkbox()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup_limits"/>
  <buttongroup name="buttonGroup_mode"/>
  <buttongroup name="buttonGroup_jump"/>
 </buttongroups>
</ui>
